bedrock/firefox/templates/firefox/features/pdf-editor-fr.html

```diff
@@ -0,0 +1,105 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/features/base-article.html" %}
+
+
+{% block article_title_short %}Comment modifier un PDF en ligne{% endblock %}
+{% block article_title %}Comment modifier un PDF en ligne et en toute sécurité{% endblock %}
+{% block page_desc %}Apprenez à modifier vos documents PDF avec Firefox à travers un guide détaillant les fonctionnalités pour ajouter du texte et même de signer vos documents{% endblock %}
+
+{% block article_content %}
+
+<p>Ah, cette frustration du quotidien qu’on connaît trop bien :  modifier un PDF pour le travail ou les études, mais avoir à imprimer sur papier, écrire à la main, et le scanner de nouveau vers votre ordinateur.</p>
+
+<p>Des options en ligne, il y en a plein. Mais est-ce totalement sécurisé ? Ces outils vous invitent à ouvrir votre document sur leur site et à le modifier dans une plateforme inconnue, avec peu de détails sur la protection de vos données. </p>
+
+<p>Après tout, fournir des informations fiscales, financières, ou personnelles, revient à partager des données sensibles à une entreprise extérieure qui pourrait de ne pas disposer de la sécurité suffisante pour protéger vos données.</p>
+
+<p>Avec <a href="{{ url('firefox.features.pdf-free') }}">l’éditeur PDF</a> Firefox, fini de se retrouver sur des interfaces peu sécurisées, chargées de pubs ou encore télécharger des logiciels encombrants. </p>
+
+<p><a href="{{ url('firefox.features.pdf-complete') }}">Remplir un PDF</a>, le modifier ou l'enregistrer n’a jamais été aussi simple et rapide. Qu'il s'agisse d'une déclaration d’impôts, d’un contrat à signer ou de documents de travail, il suffit de quelques clics pour remplir un PDF&nbsp;!</p>
+
+<h2>Modifier un PDF en ligne rapidement et facilement</h2>
+
+<ol class="mzp-u-list-styled">
+  <li>
+    <p>Il suffit d’ouvrir le PDF à remplir directement dans Firefox. Retrouver votre PDF sur votre ordinateur, de faire un clic droit dessus et de sélectionner "Ouvrir avec Firefox".</p>
```

Comment:
**Nit:** It seems you're combining point 1 and point 2 from the copy into this one list item?

![Screenshot 2024-10-09 at 10 02 24 AM](https://github.com/user-attachments/assets/a756bc8a-5d9a-4bc6-8a6e-4eb085a63787)

Reply:
✅

Reply:
**Nit:** This still needs to be separated